flink-core/src/main/java/org/apache/flink/api/common/ExecutionConfig.java

```diff
@@ -237,53 +236,26 @@ public ExecutionConfig setParallelism(int parallelism) {
 	}
 
 	/**
-	 * Gets the number of times the system will try to re-execute failed tasks. A value
-	 * of {@code -1} indicates that the system default value (as defined in the configuration)
-	 * should be used.
+	 * Sets the restart strategy configuration which defines which restart strategy shall be used
+	 * for the execution graph of the corresponding job.
```

Comment:
I would add a `<code></code>` example showing the `RestartStrategies`, which will be the common way to configure it I guess.

The text could maybe also be simplified at the end by removing execution graph and corresponding job. The average user will not know what it is. On the other hand, it might be a good pointer for someone who wants to work on it.